src/install-update.ts

```diff
@@ -0,0 +1,167 @@
+import { checkForSDKUpdates, Release } from "./check-update.ts";
+import { getJSON } from "./utilities.ts";
+import { projectScripts } from "./mod.ts";
+
+export const SDK_NAME = "the Slack SDK";
+
+interface InstallUpdateResponse {
+  name: string;
+  updates: Update[];
+  error?: {
+    message: string;
+  } | null;
+}
+
+export interface Update {
+  name: string;
+  current: string;
+  latest: string;
+  error?: {
+    message: string;
+  } | null;
+}
+
+/**
+ * updateDependencies checks for SDK-related dependency updates. If
+ * updatable releases are found, dependency files are updated with the
+ * latest dependency versions and the project changes are cached.
+ */
+export const updateDependencies = async () => {
+  const { releases } = await checkForSDKUpdates();
+  const updatableReleases = releases.filter((r) => r.current && r.latest);
+  const updateResp = await createUpdateResp(updatableReleases);
+
+  // If no errors occurred during installation, re-build
+  // project as a means to cache the changes
+  if (!updateResp.error) runBuildHook();
```

Comment:
**Open question: when should we be re-building the project and under what circumstances?** Here I've opted to only build if no errors occurred in any of the dependency updates, so as to give the user the ability to check and make adjustments themselves, but I'm not certain this is the best approach.

Reply:
This seems fine to me